### ISSUE TITLE
The Chinese path cannot be used when using the hash command.File does not exist.

### ISSUE THREAD

**Issue by masx200** [microsoft/winget-cli#315]
# Brief description of your issue

The Chinese path cannot be used when using the hash command.File does not exist.

# Steps to reproduce

```
ls D:\迅雷下载\winrar-x64-590.exe
```

```
    目录: D:\迅雷下载


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        2020/5/24     18:23        3226248 winrar-x64-590.exe

```

```
 winget hash D:\迅雷下载\winrar-x64-590.exe

```
# Expected behavior

```
 winget hash C:\Users\ma\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\AC\INetCache\KXK2CX9E\winrar-x64-590[1].exe
```
```
File Hash: af32c365c56b8a107af335fcb471298335a708d7ff803025e2b17b207c3d83ab
```
# Actual behavior

```
File does not exist: D:\杩呴浄涓嬭浇\winrar-x64-590.exe
```

# Environment

```none
[winget --info]
Windows Package Manager version
Windows: Windows.Desktop version
Package: Microsoft.DesktopAppInstaller version

Any other software?
Windows Package Manager v0.1.41331 预览
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.18363.836
Package: Microsoft.DesktopAppInstaller v1.0.41331.0

Links:
  Privacy Statement: https://aka.ms/winget-privacy
  License agreement: https://aka.ms/winget-license
  3rd Party Notices: https://aka.ms/winget-3rdPartyNotice
  Homepage:          https://aka.ms/winget
```

**Comment by yao-msft** [comment]
Related to #181

**Comment by yao-msft** [comment]
The fix should have been included with release [v0.2.2521](https://github.com/microsoft/winget-cli/releases/tag/v.0.2.2521-preview). Below is a working snapshot.

![Capture2](https://user-images.githubusercontent.com/50888816/94089120-22a85d80-fdc7-11ea-9cb2-1f141f034022.JPG)

Please feel free to reopen the issue if the fix does not work as expected.
